src/DocumentFormat.OpenXml/Packaging/PresentationDocument.cs

```diff
@@ -310,29 +309,25 @@ public DigitalSignatureOriginPart AddDigitalSignatureOriginPart()
         }
 
         /// <summary>
-        /// Adds a ThumbnailPart to the PresentationDocument.
+        /// Adds a ThumbnailPart to the WordprocessingDocument
```

Comment:
Looks like a copy/paste error.

Why are these not enabled for the interface? You may need to relax the generic constraints to enable this (i.e. these aren't parts)

Reply:
because the are package type parts which is why they weren't generating add*part methods previously as well. I decided to leave them as is because my first attempt to pull them (spreadsheetdocument, wordprocessingdocument and presentationdocument) into the new generated code met issues. I can make another attempt with the partwriter.

Reply:
ah, yeah, the methods were manually added and that's what I was referring to and why I figured there was a reason they didn't try to generate for the package parts. But I see what you mean, I'll add the interface derivation in the handwritten part cs. thanks.

Reply:
brilliant! ;)